src/status_im/data_store/realm/schemas/account/v8/browser.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im.data-store.realm.schemas.account.v8.browser)
+
+(def schema {:name       :browser
+             :primaryKey :browser-id
+             :properties {:browser-id :string
+                          :name       :string
+                          :timestamp  :int
+                          :dapp?      {:type    :bool
+                                       :default false}
+                          :url        {:type     :string
+                                       :optional true}
+                          :contact    {:type     :string
+                                       :optional true}
+                          :history-index  {:type :int
+                                           :optional true
+                                           :default 0}}
+             :history    {:type     :vector
+                          :optional true
+                          :default (vector "")}})
```

Comment:
`[""]` looks better to me, but I'm ok with `(vector "")`, just to make sure, why do you need it to be initialized with a vector with an empty string?

Reply:
No longer needed, used to be necessary to prevent erroring out. Will remove, same for the `:history-index` default. :+1:

Reply:
Done!